docs/conf.py

```diff
@@ -32,6 +32,16 @@
     'show-inheritance': True
 }
 
+mathjax3_config = {
+    'tex': {
+        'macros': {
+            'diag': ['\\operatorname{diag}']
+        }
+    }
+}
+
+myst_heading_anchors = 2
```

Comment:
What does this do?

Reply:
Fixes an unrelated warning from the contributing guide markdown